app/models/wpcc/salary_frequency.rb

```diff
@@ -0,0 +1,27 @@
+module Wpcc
+  class SalaryFrequency
+    include ActiveModel::Model
+    attr_accessor :params_salary_frequency, :session_salary_frequency
+
+    DEFAULT_SALARY_FREQUENCIES = Hash[
+      Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map do |elem|
+        [elem, elem == 'year' ? 'month' : elem]
```

Comment:
@margOnline yes I did. I then noticed that the same piece of code is replicated in another file. I thought there were some suggestions by @tomas-stefano. They were excellent suggestions which I thought would be implemented. 

First suggestion
```
Wpcc::YourDetailsForm::SALARY_FREQUENCIES.tap do |frequencies|
  frequencies[frequencies.index('year')] = 'month'
end
```

Second suggestion
```
Hash[
   Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map { |e| [e, e] }].tap do |hash|
     hash['year'] = 'month'
end
```